docs/current_docs/api/arguments.mdx

```diff
@@ -490,6 +545,19 @@ Here's an example of a Dagger Function with an optional argument:
 ```typescript file=./snippets/functions/arguments-optional/typescript/index.ts
 ```
 </TabItem>
+<TabItem value="PHP">
+:::note
+The definition of _optional_ varies between Dagger and PHP.
```

Comment:
```suggestion
The definition of optional varies between Dagger and PHP.
```
For style consistency when multiple authors are involved, we generally try to avoid using formatting marks like italics, bold, and so on.